src/xrCore/_math.cpp

```diff
@@ -201,6 +201,58 @@ XRCORE_API u32 GetTicks()
 {
     return SDL_GetTicks();
 }
+
+XRCORE_API size_t GetThreadsCounts() noexcept
+{
+#if defined(XR_PLATFORM_WINDOWS)
+    DWORD length = 0;
+    size_t concurrency = 0; 
+    const auto validConcurrency = [&concurrency]() noexcept -> size_t {
+        if (concurrency == 0)
+            return std::thread::hardware_concurrency();
+        else
+            return concurrency;
```

Comment:
```suggestion
        return concurrency == 0 ? std::thread::hardware_concurrency() : concurrency;
```

Reply:
fix it https://github.com/OpenXRay/xray-16/pull/1514/commits/7dc09604acc59df42260a0cb11c58d4b03901a74